azerothcore/azerothcore-wotlk
# fix(DB/Spell): Giant Softshell drops

Giant Softshell Clam's should only be lootable with a certain quest (6142)



## Changes Proposed:
-  data_8 (questid) has been changed on the [Giant Softshell Clam](https://wotlk-twinhead.twinstar.cz/?object=177784) to [Clam Bait](https://wotlk-twinhead.twinstar.cz/?quest=6142)

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/5585

## SOURCE:

 [Soft-shelled Clam](https://wotlk-twinhead.twinstar.cz/?item=15874) is a Quest item required for [Clam Bait](https://wotlk-twinhead.twinstar.cz/?quest=6142) 

## Tests Performed:

- I tested both test cases below on my local machine/server and they work as intended.


## How to Test the Changes:


### Test case 1
1. `.quest add 6142`
2. `.go o 31973`
3. Open the clam.
4. Check if you got a Soft-shelled Clam.

### Test case 2
1. Do step 2 and 3 of test case 1 and see if its not interactable.

## Known Issues and TODO List:


Nothing.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
